spec/api-remote-spec.js

```diff
@@ -94,41 +93,41 @@ describe('remote module', () => {
 
     it('should work with function properties', () => {
       let a = remote.require(path.join(fixtures, 'module', 'export-function-with-properties.js'))
-      assert.strictEqual(typeof a, 'function')
-      assert.strictEqual(a.bar, 'baz')
+      expect(a).to.be.a('function')
+      expect(a.bar).to.equal('baz')
 
       a = remote.require(path.join(fixtures, 'module', 'function-with-properties.js'))
-      assert.strictEqual(typeof a, 'object')
-      assert.strictEqual(a.foo(), 'hello')
-      assert.strictEqual(a.foo.bar, 'baz')
-      assert.strictEqual(a.foo.nested.prop, 'yes')
-      assert.strictEqual(a.foo.method1(), 'world')
-      assert.strictEqual(a.foo.method1.prop1(), 123)
+      expect(a).to.be.an('object')
+      expect(a.foo()).to.equal('hello')
+      expect(a.foo.bar).to.equal('baz')
+      expect(a.foo.nested.prop).to.equal('yes')
+      expect(a.foo.method1()).to.equal('world')
+      expect(a.foo.method1.prop1()).to.equal(123)
 
-      assert.ok(Object.keys(a.foo).includes('bar'))
-      assert.ok(Object.keys(a.foo).includes('nested'))
-      assert.ok(Object.keys(a.foo).includes('method1'))
+      expect(Object.keys(a.foo).includes('bar')).to.be.true()
```

Comment:
these can be: `expect(obj).to.have.key('some_key')`

Reply:
for some reason it does not work, I am getting
AssertionError: expected [Function: remoteMemberFunction] to have key 'bar'

Reply:
this does work `expect(a.foo).to.have.a.property('bar')`